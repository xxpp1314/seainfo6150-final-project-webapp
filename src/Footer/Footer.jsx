import React from 'react';
import styles from "./Footer.module.css";
import logo from "../images/logo.png";
import { Link } from "react-router-dom";

const Footer = () => {
    return (
        <div className={styles.footer}>
            <div className={styles.margin}>
            <img className={styles.logo} src={logo} alt=""/>
                <li className={styles.li}>
                    <p className={styles.p}> Pet Home Inc.</p>
                </li>
                <Link to="/contactUs" className={styles.optionLink}>
                <p className={styles.p}>Contact Us</p>
                </Link>
                <Link to="/feedback" className={styles.optionLink}>
                <p className={styles.p}>Feedback</p>
                </Link>
                <li className={styles.li}></li>
                    <p className={styles.p}>Open Hours: Mon - Thu: 9 a.m. - 8 p.m. Fri - Sun: 10 a.m. - 7 p.m.</p>
            </div>
        </div>
    )
}

export default Footer;