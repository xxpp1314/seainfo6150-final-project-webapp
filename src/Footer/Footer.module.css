.footer{
    padding-top: 3%;
    text-align: center;
    color:rgb(117, 104, 104);
    background-image: linear-gradient(to right,rgb(83, 15, 15), rgb(49, 12, 25) 70%);
    font-family: 'Space Grotesk', sans-serif;
    font-size: smaller;
}

.margin{
    border-top: 0.5px solid rgb(102, 18, 18);
    padding-top:0.5%;
}

.p {
    color: white;
    font-size: 100%;
}

.li {
    display: inline;
    margin: 0;
    padding: 0;
}

.logo{
    border-radius: 25%;
}

.optionLink {
    text-decoration-line: none;
    color: white;
}

@media screen and (min-width: 400px) {
    .logo {
      width: 30px;
    }
    .title {
      font-size: 12px;
    }
    .navTab {
      margin-left: 20px;
      font-size: 15px;
    }
    .subscribe {
      padding: 1px 5px;
      font-size: 12px;
    }
  }

  @media screen and (min-width: 600px) {
    .logo {
      width: 40px;
    }
    .title {
      font-size: 15px;
    }
    .navTab {
      margin-left: 20px;
      font-size: 15px;
    }
    .subscribe {
      padding: 2px 10px;
      font-size: 15px;
    }
  }
  
  @media screen and (min-width: 900px) {
    .logo {
      width: 60px;
    }
    .title {
      font-size: 20px;
    }
    .navTab {
      margin-left: 50px;
      font-size: 18px;
    }
    .subscribe {
      padding: 5px 20px;
      font-size: 18px;
    }
  }
  
  @media screen and (min-width: 1200px) {
    .logo {
      width: 80px;
    }
    .title {
      font-size: 25px;
    }
    .navTab {
      margin-left: 80px;
      font-size: 20px;
    }
    .subscribe {
      padding: 5px 25px;
      font-size: 20px;
    }
  }