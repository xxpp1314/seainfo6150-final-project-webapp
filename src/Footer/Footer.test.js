import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer.jsx";
import { BrowserRouter } from 'react-router-dom';

it("renders the Footer component correctly", () => {
  const { container } = render(
    <BrowserRouter>  
        <Footer/>
    </BrowserRouter> 
  );
  expect(container).toMatchSnapshot();
});
