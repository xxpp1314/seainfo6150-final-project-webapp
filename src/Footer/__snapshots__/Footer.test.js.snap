// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Footer component correctly 1`] = `
<div>
  <div
    class="footer"
  >
    <div
      class="margin"
    >
      <img
        alt=""
        class="logo"
        src="logo.png"
      />
      <li
        class="li"
      >
        <p
          class="p"
        >
           Pet Home Inc.
        </p>
      </li>
      <a
        class="optionLink"
        href="/contactUs"
      >
        <p
          class="p"
        >
          Contact Us
        </p>
      </a>
      <a
        class="optionLink"
        href="/feedback"
      >
        <p
          class="p"
        >
          Feedback
        </p>
      </a>
      <li
        class="li"
      />
      <p
        class="p"
      >
        Open Hours: Mon - Thu: 9 a.m. - 8 p.m. Fri - Sun: 10 a.m. - 7 p.m.
      </p>
    </div>
  </div>
</div>
`;
